src/main/java/frc/robot/subsystems/Manager.java

```diff
@@ -0,0 +1,202 @@
+                centerNoteTimer.stop();
+                centerNoteTimer.reset();
+            }
+        } else if (state == ManagerStates.PULL_IN) {
+            stateString = "Pull In";
+            centerNoteTimer.start();
+            intake.setState(IntakeStates.PULL_IN);
+            shooter.setState(ShootingStates.REVERSING);
+          
+            if (centerNoteTimer.get() > Constants.Shooter.PULL_CENTER_NOTE_TIME) {
+                reset();
+                state = ManagerStates.IDLE;
+                centerNoteTimer.stop();
+                centerNoteTimer.reset();
+            }
+        } else if (state == ManagerStates.WAIT_FOR_BACK) {
+            centerNoteTimer.start();
+            stateString = "Wait For Back";
+            intake.setState(IntakeStates.OFF);
+            if (centerNoteTimer.get() > Constants.Shooter.RETURN_CENTER_NOTE_TIME) {
+                reset();
+                state = ManagerStates.PUSH_OUT;
+                centerNoteTimer.stop();
+                centerNoteTimer.reset();
+            }
+        }
+         else if (state == ManagerStates.INTAKING) {
+            intake.setState(IntakeStates.INTAKING);
+            shooter.setState(ShootingStates.OFF);
+            stateString = "Intaking";
+            if ((intake.intakeMotor.getSupplyCurrent().getValueAsDouble() > 30 && !DriverStation.isAutonomous()) || robot.controller.getBButtonPressed()) { // Current sensing to detect when we have the note.
+                System.out.println("Current sensing made it go in");
+                state = ManagerStates.IDLE;
+                reset();
+            } else if (robot.controller.getRightBumper()) {
+                state = ManagerStates.OUTTAKING;
+                reset();
+            }
+        } else if (state == ManagerStates.OUTTAKING) {
+            intake.setState(IntakeStates.OUTTAKING);
+            shooter.setState(ShootingStates.OFF);
+            if (robot.controller.getRightBumperReleased()) {
+                state = ManagerStates.INTAKING;
+                reset();
+            }
+            stateString = "Outtaking";
+        } else if (state == ManagerStates.SHOOTING) {
+            shooter.setState(ShootingStates.SHOOTING);
+            shooterTimer.start();
+            intake.setState(IntakeStates.FEEDING);
+            if (shooterTimer.get() > Constants.Shooter.SHOOTER_TIME) {
+                shooterTimer.stop();
+                shooterTimer.reset();
+                state = ManagerStates.IDLE;
+                reset();
+            }
+              
+            stateString = "Shooting";
+        } else if (state == ManagerStates.SCORING_AMP) {
+            shooter.setState(ShootingStates.OFF);
+            intake.setState(IntakeStates.GOING_TO_AMP);
+            if (intake.nearSetpoint()) {
+                intake.setState(IntakeStates.AMP_SCORING);
+                shooterTimer.start();
+                if (shooterTimer.get() > 1) {
```

Comment:
Didn't someone put this to Constants? Was it not committed? Was it lost to the void? Track this down